### PR TITLE
Feature(Chart): Chaos charts for Docker Service Kill experiment

### PR DESCRIPTION
**Pull request by ghoshankur1983**
Signed-off-by: Ankur Ghosh ( ankur.ghosh3@wipro.com)

Adding the Chaos Charts for the Docker Service Kill experiment.

### CODE REVIEW

File: charts/generic/docker-service-kill/docker-service-kill.chartserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  version: 0.1.0
+  annotations:
+    categories: docker-service-kill
+    repository: https://github.com/litmuschaos/litmus-ansible/tree/master/generic/docker-service-kill
+    support: https://kubernetes.slack.com/messages/CNXNB0ZTN
+spec:
+  displayName: docker-service-kill 
+  categoryDescription: >
+    Randomly stops Docker services temporarily 
+  keywords: ['pods', 'kubernetes', 'sample-category', 'nginx']
```
Comment (ispeakc0de) [inline charts/generic/docker-service-kill/docker-service-kill.chartserviceversion.yaml]: can we remove sample-category?

File: charts/generic/docker-service-kill/rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  name: docker-service-kill-sa
+  namespace: default
+  labels:
+    name: docker-service-kill-sa
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: docker-service-kill-sa
+  namespace: default
```
Comment (ispeakc0de) [inline charts/generic/docker-service-kill/rbac.yaml]: no namespace field required for clusterrole

File: charts/generic/docker-service-kill/rbac.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  verbs: ["create","list","get","patch","update","delete"]
+- apiGroups: [""]
+  resources: ["nodes"]
+  verbs: ["get","list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: docker-service-kill-sa
+  namespace: default
```
Comment (ispeakc0de) [inline charts/generic/docker-service-kill/rbac.yaml]: same as above

**Review by ispeakc0de — CHANGES_REQUESTED** [review]

**Comment by ghoshankur1983** [comment]
Incorporated the changes. Please close the pull request if everything is fine.

**Review by uditgaurav — APPROVED** [review]
lgtm

**Review by ispeakc0de — APPROVED** [review]

**Comment by uditgaurav** [comment]
We can add icons and a Wipro card in upcoming PRs.

**Pull request merged**